### DISCUSSION TITLE
building problem

### DISCUSSION THREAD

**Discussion by advance-hub** [vuejs/vitepress#4259] [Q&A]
![image](https://github.com/user-attachments/assets/e49a24af-980e-4818-99ed-d64b1692e45f)
There is no tmp under.vitepress folder. Now there is always a problem with packaging, and I don't know why

**Comment by brc-dd** [comment]
try changing the build command to this in your package.json:

```sh
DEBUG=true vitepress build docs
```

.temp is deleted after build when you don't specify DEBUG. But the issue here is something else.
